### PR TITLE
fix docker-compose to work with temporal docker images

### PR DESCRIPTION
**Pull request by samarabbas**
- Docker compose fixes to use grpc ports
- Docker compose fixes to use Temporal images from dockerhub

### CODE REVIEW

File: docker/buildkite/docker-compose-local.yml [hunk 1]
```diff
@@ -81,10 +81,14 @@ services:
...
     ports:
-      - "7933:7933"
-      - "7934:7934"
-      - "7935:7935"
-      - "7939:7939"
+      - "7233:7233"
+      - "7234:7234"
+      - "7235:7235"
+      - "7239:7239"
+      - "6933:6933"
```
Comment (alexshtin) [inline docker/buildkite/docker-compose-local.yml]: What are these `6xxx`?

File: docker/docker-compose-es.yml [hunk 2]
```diff
@@ -53,9 +57,9 @@ services:
   cadence-web:
     image: ubercadence/web:3.4.1
     environment:
-      - "CADENCE_TCHANNEL_PEERS=cadence:7933"
+      - "CADENCE_TCHANNEL_PEERS=temporal:7233"
```
Comment (alexshtin) [inline docker/docker-compose-es.yml]: This won't work anyway :-)

File: docker/buildkite/docker-compose-local.yml (hunk 1 above)
Comment (samarabbas) [reply docker/buildkite/docker-compose-local.yml]: They are the ringpop ports.

File: docker/docker-compose-es.yml (hunk 2 above)
Comment (samarabbas) [reply docker/docker-compose-es.yml]: I know.  That is next on my list.

**Pull request merged**